src/index.ts

```diff
@@ -1,77 +1,83 @@
 import '@k2oss/k2-broker-core';
 
 metadata = {
-    systemName: "com.sample.odata",
+    systemName: "sampleOdata",
     displayName: "Sample OData Broker",
     description: "An sample broker that accesses OData."
 };
 
-ondescribe = function() {
+ondescribe = async function (): Promise<void> {
     postSchema({
         // https://www.odata.org/getting-started/understand-odata-in-6-steps/
         // http://olingo.apache.org/doc/javascript/index.html
         objects: {
-            "com.sample.odata.people": {
+            "people": {
                 displayName: "People",
                 description: "People",
                 properties: {
-                    "com.sample.odata.people.userName": {
+                    "userName": {
                         displayName: "User Name",
-                        type: "string" 
+                        type: "string"
                     },
-                    "com.sample.odata.people.city": {
+                    "city": {
                         displayName: "City",
-                        type: "string" 
+                        type: "string"
                     }
                 },
                 methods: {
-                    "com.sample.odata.people.get": {
+                    "get": {
                         displayName: "Get People",
                         type: "read",
-                        outputs: [ "com.sample.odata.people.userName", "com.sample.odata.people.city" ]
+                        outputs: ["userName", "city"]
                     }
                 }
             }
         }
     });
 }
 
-onexecute = function(objectName, methodName, parameters, properties) {
-    switch (objectName)
-    {
-        case "com.sample.odata.people": onexecuteGet(methodName, parameters, properties); break;
+onexecute = async function (objectName, methodName, parameters, properties): Promise<void> {
+    switch (objectName) {
+        case "people": onexecuteGet(methodName, parameters, properties); break;
```

Comment:
doesn't this need to be awaited?

Reply:
And can you remove the dots everywhere? In this example, com.sample.odata.people should just be people.

Reply:
@michta42 It is removed. The Red color is old code. The green color is changed code

Reply:
@ryanelfmank2 added await